wagtail/admin/templates/wagtailadmin/edit_handlers/tabbed_interface.html

```diff
@@ -1,8 +1,8 @@
 {% load wagtailadmin_tags %}
-<ul class="tab-nav merged" role="tablist">
+<ul class="tab-nav merged" role="tablist" aria-owns="{% for child in self.children %}tab-label-{{ child.heading|cautious_slugify }} {% endfor %}">
```

Comment:
We had a quick chat about potentially turning this into `div` with the `a` right within the container – I see that in the end you’ve opted for `aria-owns`, does that work better / equally as well?

Reply:
Restructuring the CSS was going to be a lot of work, so I tried to kick that can down the road. 😝 As far as I can tell `aria-owns` works equally well, but it has more template writing overhead, of course. I could be open to being pressured into restructuring the HTML and CSS here, if you wanted a more thorough PR.

Reply:
Cool. I’ll need to check what the recommendations are for tab elements, basically wondering whether lists are appropriate to start with in tab markup.